pkg/repository/v1/sqlcv1/rate_limits.sql

```diff
@@ -0,0 +1,171 @@
+-- name: UpsertRateLimit :one
+INSERT INTO "RateLimit" (
+    "tenantId",
+    "key",
+    "limitValue",
+    "value",
+    "window"
+) VALUES (
+    @tenantId::uuid,
+    @key::text,
+    sqlc.arg('limit')::int,
+    sqlc.arg('limit')::int,
```

Comment:
is this a legacy thing? I think I've seen this in the python code too

Reply:
This was from the old engine/API code that was copy-pasted, removed it since we're not using it.

Reply:
sweet 👍